packages/core/src/codewhisperer/util/customizationUtil.ts

```diff
@@ -147,6 +147,9 @@ export const setSelectedCustomization = async (customization: Customization, isO
     }
     vsCodeState.isFreeTierLimitReached = false
     await Commands.tryExecute('aws.amazonq.refreshStatusBar')
+
+    // hack: triggers amazon q to send the customizations back to flare
+    await Commands.tryExecute('aws.amazonq.updateCustomizations')
```

Comment:
Why not call a function instead of a command?

Commands should be avoided, because they are less reliable and less visible if they fail (partly because they tend to be async).